scripts/deploy-dev-ec2.sh

```
@@ -0,0 +1,187 @@
+./aws/install
+rm -rf aws awscliv2.zip
+EOF
+
+# --- Launch EC2 Instance ---
+echo "🚀 Launching EC2 instance... (This may take a moment)"
+if ! aws ec2 describe-instances --filters "Name=tag:Name,Values=${PROJECT_NAME}-instance-${ENVIRONMENT}" "Name=instance-state-name,Values=running" --query "Reservations[].Instances[]" --output text | grep -q .; then
+  INSTANCE_ID=$(aws ec2 run-instances \
+    --image-id "$AMI_ID" \
+    --instance-type "$INSTANCE_TYPE" \
+    --key-name "$KEY_NAME" \
+    --security-group-ids $(aws ec2 describe-security-groups --group-names "$SECURITY_GROUP_NAME" --query "SecurityGroups[0].GroupId" --output text) \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)